examples/oracle.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# This file is part of pycloudlib. See LICENSE file for license information.
+"""Basic examples of various lifecycle with an OCI instance."""
+
+import logging
+import sys
+from base64 import b64encode
+
+import pycloudlib
+
+cloud_config = """#cloud-config
+runcmd:
+  - echo 'hello' > /home/ubuntu/example.txt
+"""
+
+
+def demo(availability_domain: str, compartment_id: str):
```

Comment:
The default way of getting these values is from the `~/.config/pycloudlib.toml` values. If we default these to `None` and no arguments are passed to this script, it should "just work" if the toml values were set correctly.

Reply:
Great point! I'll modify the script to do so. I thank _that_ truly makes the script more streamlined compared to the other bloated changes i added. thanks!